packages/babel-plugin-minify-constant-folding/src/replacements.js

```diff
@@ -0,0 +1,123 @@
+const other = Symbol("other");
```

Comment:
@boopathi It is called when the mapping object doesn’t provide a handler for the specified key. For example, the `Array` handler uses it to return the value at the specified index if the value can be coerced to an integer.